packages/app/src/pages/profile/Profile.module.css

```diff
@@ -0,0 +1,4 @@
+.profile__center {
```

Comment:
I'd decribe the element more precise instead of just the name, like: `.profile__container`. This is much clearer. Please also consider it at other positions :-)

Reply:
I changed it here and on other positions. Sometimes we are using compontent__card. That is fine or should I also change these?